pkg/kor/kor.go

```diff
@@ -56,17 +61,29 @@ func GetKubeClient(kubeconfig string) *kubernetes.Clientset {
 	return clientset
 }
 
-func SetNamespaceList(namespace string, kubeClient *kubernetes.Clientset) []string {
-	var namespaces []string
-	if namespace != "" {
-		namespaces = append(namespaces, namespace)
-	} else {
-		namespaceList, err := kubeClient.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to retrieve namespaces: %v\n", err)
-			os.Exit(1)
+func SetNamespaceList(lists IncludeExcludeLists, kubeClient *kubernetes.Clientset) []string {
```

Comment:
Can we change `lists IncludeExcludeLists`
to 
`namespaceLists IncludeExcludeLists`

Reply:
done